js/modules/k6/http/http_request.go

```diff
@@ -137,16 +137,38 @@ func (h *HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.Stat
 		req.Header.Set("User-Agent", userAgent.String)
 	}
 
-	tags := map[string]string{
-		"proto":  "",
-		"status": "0",
-		"method": method,
-		"url":    url.URLString,
-		"name":   url.Name,
-		"group":  state.Group.Path,
-		"vu":     strconv.FormatInt(state.Vu, 10),
-		"iter":   strconv.FormatInt(state.Iteration, 10),
+	var tags map[string]string
+	if state.Options.DefaultTags == nil {
+		tags = map[string]string{
+			"proto":  "",
+			"status": "0",
+			"method": method,
+			"url":    url.URLString,
+			"name":   url.Name,
+			"group":  state.Group.Path,
+		}
+	} else {
+		tags = map[string]string{}
+		if state.Options.DefaultTags["proto"] {
+			tags["proto"] = ""
+		}
+		if state.Options.DefaultTags["status"] {
+			tags["status"] = "0"
```

Comment:
Same question here for `proto` and `status`, and below for the websockets - why is this needed?

Reply:
Agree, see comment below.